sky/utils/kubernetes/k8s_gpu_labeler_setup.yaml

```diff
@@ -108,9 +108,13 @@ data:
                     labelled = True
                     break
             if not labelled:
-                # If we didn't find a canonical name, just use the GPU name
-                # with spaces replaced by dashes
-                gpu_label = gpu_name.lower().replace(' ', '-')
+                # If we didn't find a canonical name:
+                # 1. remove 'NVIDIA ' if present (e.g., 'NVIDIA RTX A6000' -> 'RTX A6000')
+                # 2. remove 'GeForce ' if present (e.g., 'NVIDIA GeForce RTX 3070' -> 'RTX 3070')
+                # 3. replace 'RTX ' with 'RTX' (without spaces) (e.g., 'RTX 6000' -> 'RTX6000')
+                # 4. replace any other spaces with dashes (e.g. 'RTX 2080 Ti' -> 'RTX2080-Ti')
+                gpu_name = gpu_name.lower().replace('nvidia ', '').replace('geforce ', '').replace('rtx ', 'rtx').replace(' ', '-')
```

Comment:
Generally kubernetes labels and values are lowercase, so keeping in line with convention. Also since label matching is case-sensitive, changing now would require deeper backward compatibility changes